include/seqan3/alignment/all.hpp

```diff
@@ -4,6 +4,8 @@
 // This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
 // shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
 // -----------------------------------------------------------------------------------------------------
+// -----------------------------------------------------------------------------------------------------
+// -----------------------------------------------------------------------------------------------------
```

Comment:
seems like your script is very happily adding line comments :D

Reply:
:facepalm: yes! I fixed that like twice yesterday ;-), but at the end I still got it wrong, very weird.